package.json

```diff
@@ -9,6 +9,9 @@
   },
   "author": "",
   "license": "AGPL-3.0",
+  "scripts": {
+    "build": "make npm-init build-js-production bundle-simplewebrtc"
```

Comment:
What if make is not available? :)

Reply:
For now this is going to be used only with /compile bot, so... please do not remove `make` from the Docker image of the bot :-P

Reply:
I have no idea if it's even there :p

Reply:
I do, it is ;-) So as long as you do not add `apt-get remove make` to the _Dockerfile_ we are safe :-P

Reply:
Perfect then!!